### DISCUSSION TITLE
Using revChatGPT V1 today has been reporting an error: OpenAI: (code: 524) 。What is the reason?

### DISCUSSION THREAD

**Discussion by luluhuhu** [acheong08/ChatGPT#1416] [Q&A]
Using revChatGPT V1 today has been reporting an error: OpenAI: (code: 524)
What is the reason?

**Comment by ggg5438** [comment]
I got the same error

**Comment by codelauncher444** [comment]
Did it get solved?

**Reply by luluhuhu** [reply]
not yet resolved
But I have no problem running on the local mac, but occasionally report 403

**Comment by luluhuhu** [comment]
524 error is not reported now,
The program automatically becomes better,
It is suspected that it is caused by the response timeout of the source site. The source site has too many processing tasks. I hope the author can optimize revChatGPT
@acheong08

**Comment by gchust** [comment]
https://github.com/acheong08/ChatGPT/issues/1415
